app/src/main/res/values-de/strings.xml

```diff
@@ -185,6 +187,7 @@
     <string name="none">Keine(r)</string>
     <string name="section_issue_labels">Markierungen:</string>
     <string name="log_in">Einloggen</string>
+    <string name="log_out">Ausloggen</string>
```

Comment:
Than it is right. 
But normaly "signup" is equal to "Registrieren" and "log_in" "Anmelden".
But we have signup always translated as "sign up" also "Anmelden". We shouldn't break that.

So Ausloggen is ok ;)